backend/api/migrations/0001_initial.py

```
@@ -8,18 +11,141 @@ class Migration(migrations.Migration):
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("password", models.CharField(max_length=128, verbose_name="password")),
+                (
+                    "last_login",
+                    models.DateTimeField(
+                        blank=True, null=True, verbose_name="last login"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)